src/django_components/constants.py

```diff
@@ -0,0 +1,3 @@
+UID_LENGTH = 6
```

Comment:
Defined the ID length and the prefix here in `constants.py`, so that, if we ever need to change the prefix or the length, it would be only a matter of changing these and updating the tests.